### PR TITLE
GSoC 2022: Add German translation

### PR DESCRIPTION
**Pull request by Echolon**
I don't know how to link this correctly

### CODE REVIEW

File: content/community/gsoc-2022.de.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+- [Twitter](https://twitter.com/xmpp)
+- [Mastodon/Fediverse](https://fosstodon.org/@xmpp/)
+- [Youtube](https://www.youtube.com/c/XMPPStandardsFoundation)
+
+Wir freuen uns auf diesen Ausblick und kommt auf uns zu!
+ - XSF Organisation Admin
+
+### Quellen
+
+- Google Summer of Code, Wikipedia contributors, Wikipedia, The Free Encyclopedia, 22 November 2021 23:03 UTC, 22 January 2022 14:31 UTC, https://en.wikipedia.org/w/index.php?title=Google_Summer_of_Code&oldid=1056637774, revision history statistics, 1056637774
```
Comment (wurstsalat3000) [inline content/community/gsoc-2022.de.md:42]: Please update these links

**Comment by wurstsalat3000** [comment]
Translation works fine, but i18n is not yet integrated for pages. This PR can be merged, but for now the page is only reachable via /de/community/gsoc-2022/ until I write the necessary template.

**Comment by Echolon** [comment]
> Translation works fine, but i18n is not yet integrated for pages. This PR can be merged, but for now the page is only reachable via /de/community/gsoc-2022/ until I write the necessary template.

Oh no, how much work would that be?

**Comment by wurstsalat3000** [comment]
Not sure yet. I'm having a look ;)

File: content/community/gsoc-2022.de.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+- [Twitter](https://twitter.com/xmpp)
+- [Mastodon/Fediverse](https://fosstodon.org/@xmpp/)
+- [Youtube](https://www.youtube.com/c/XMPPStandardsFoundation)
+
+Wir freuen uns auf diesen Ausblick und kommt auf uns zu!
+ - XSF Organisation Admin
+
+### Quellen
+
+- Google Summer of Code, Wikipedia contributors, Wikipedia, The Free Encyclopedia, 22 November 2021 23:03 UTC, 22 January 2022 14:31 UTC, [Permantent link](https://en.wikipedia.org/w/index.php?title=Google_Summer_of_Code&oldid=1056637774), revision history statistics, 1056637774
```
Comment (wurstsalat3000) [inline content/community/gsoc-2022.de.md:42]: Permantent

Comment (Echolon) [reply content/community/gsoc-2022.de.md:42]: 517474b8ddb6612d88b78e88fe1d89da22b9a5e0

**Pull request merged**